### PR TITLE
Modern theme support for tile component

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**
https://blackbaud.invisionapp.com/console/share/UG1N3XTCC7/483430271

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-tiles?_sp.spa=pr-tile-modern-theme](https://developer.blackbaud.com/skyux-tiles?_sp.spa=pr-tile-modern-theme)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-tiles?_sp.spa=pr-tile-modern-theme](https://developer.blackbaud.com/skyux-tiles?_sp.spa=pr-tile-modern-theme)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-tiles?_sp.spa=pr-tile-modern-theme](https://developer.blackbaud.com/skyux-tiles?_sp.spa=pr-tile-modern-theme)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-tiles?_sp.spa=pr-tile-modern-theme](https://developer.blackbaud.com/skyux-tiles?_sp.spa=pr-tile-modern-theme)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-tiles?_sp.spa=pr-tile-modern-theme](https://developer.blackbaud.com/skyux-tiles?_sp.spa=pr-tile-modern-theme)

**Comment by Blackbaud-ToddRoberts** [comment]
The header buttons (except for chevron) need to be updated to match borderless icon button (size is wrong, can't tell about color)

**Comment by Blackbaud-AlexKingman** [comment]
> The header buttons (except for chevron) need to be updated to match borderless icon button (size is wrong, can't tell about color)

Looks like there were some other styles overriding sizes. I've fixed that now and ensured everything has the borderless icons styles. Although, it does seem like they're not the same size - especially the grid icon. Even though they're all 16px, it seems a little of.... Is this how it's supposed to look?
Screen Shot 2021-03-18 at 2 54 40 PM

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-tiles?_sp.spa=pr-tile-modern-theme](https://developer.blackbaud.com/skyux-tiles?_sp.spa=pr-tile-modern-theme)

**Comment by Blackbaud-ToddRoberts** [comment]
I believe the drag icon has an update in the latest icons PR to make it larger, currently it does render smaller than the other icons.

**Comment by Blackbaud-ToddRoberts** [comment]
Also need 10px of margin between the buttons - with the small size, the spacing is important to allow accurate click targeting.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-tiles?_sp.spa=pr-tile-modern-theme](https://developer.blackbaud.com/skyux-tiles?_sp.spa=pr-tile-modern-theme)

**Review by Blackbaud-ToddRoberts — CHANGES_REQUESTED** [review]
Spacing between tiles needs to be updated to 30px to match modern spacing guidelines

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-tiles?_sp.spa=pr-tile-modern-theme](https://developer.blackbaud.com/skyux-tiles?_sp.spa=pr-tile-modern-theme)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-tiles?_sp.spa=pr-tile-modern-theme](https://developer.blackbaud.com/skyux-tiles?_sp.spa=pr-tile-modern-theme)

**Review by Blackbaud-ToddRoberts — APPROVED** [review]

File: src/app/public/modules/tiles/tile-dashboard/tile-dashboard.component.spec.ts [hunk 1]
```diff
@@ -20,44 +17,87 @@ import {
...
+  Tile1TestComponent
+} from './fixtures/tile1.component.fixture';
 
-import { TileDashboardOnPushTestComponent } from './fixtures/tile-dashboard-on-push.component.fixture';
+import {
+  Tile2TestComponent
+} from './fixtures/tile2.component.fixture';
 
-import { TileTestContext } from './fixtures/tile-context.fixture';
+import { TileDashboardOnPushTestComponent
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/tiles/tile-dashboard/tile-dashboard.component.spec.ts:61]: Import style.

**Review by Blackbaud-SteveBrush — COMMENTED** [review]
One small thing. Are we not disabling view encapsulation for this component?

File: src/app/public/modules/tiles/tile-dashboard/tile-dashboard.component.spec.ts (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/tiles/tile-dashboard/tile-dashboard.component.spec.ts]: Fixed.

**Comment by Blackbaud-AlexKingman** [comment]
> Are we not disabling view encapsulation for this component?

Good question. I didn't have to write any *new* styles that used `ng-deep`. I just had to overwrite some styles that were already using `ng-deep`. While tiles uses this pattern sometimes, it wasn't nearly as bad as some of the other modern upgraded. I'm on the fence about removing encapsulation, but could be persuaded.
Do you have strong feelings on it?

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-tiles?_sp.spa=pr-tile-modern-theme](https://developer.blackbaud.com/skyux-tiles?_sp.spa=pr-tile-modern-theme)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**